proto/axelarcork/v1/axelarcork.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package axelarcork.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/peggyjv/sommelier/v6/x/axelarcork/types";
+
+message AxelarCork {
+  // call body containing the ABI encoded bytes to send to the contract
+  bytes encoded_contract_call = 1;
+  // the chain ID of the evm target chain
+  uint64 chain_id = 2;
+  // address of the contract to send the call
+  string target_contract_address = 3;
+}
+
+message ScheduledAxelarCork {
+  AxelarCork   cork         = 1;
+  uint64 block_height = 2;
+  string validator    = 3;
+  bytes  id           = 4;
```

Comment:
As the file changed, noting here that raw bytes in the proto may not be how we want to represent a cork ID -- perhaps a hex string would be more appropriate.